app/mailers/cards_mailer.rb

```diff
@@ -0,0 +1,8 @@
+class CardsMailer < ActionMailer::Base
+  default from: 'info@flashcards.com'
+
+  def pending_cards_notification(email, cards_count)
+    mail(to: email,
+         subject: 'Наступила даты пересмотра карточек.')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.